docs/api/js/utils.md

```diff
@@ -553,6 +554,34 @@ export function getHashedL2ToL1Msg(sender: Address, msg: BytesLike, txNumberInBl
 }
 ```
 
+
+### `getPaymasterParams`
```

Comment:
We have this same function documented in the [Paymaster Utilities](https://era.zksync.io/docs/api/js/paymaster-utils.html#getpaymasterparams) section. Is that intentional? If so, we should update that documented Paymaster Utilities to show full implementation as this one does.

Reply:
missed that, I'll revert this change

Reply:
fixed in https://github.com/matter-labs/zksync-web-era-docs/pull/625/commits/e9a625b0b820ad302d3b0eb7e78d2fa7e6970cd1